### ISSUE TITLE
q-numeric with undefined v-model shows 1

### ISSUE THREAD

**Issue by alexey2baranov** [quasarframework/quasar#333]
Hello!

```html
<q-numeric v-model="some_variable_with_undefined_value" :min="1" :max="9999"></q-numeric>

```

this shows 1 but should nothing

**Comment by rstoenescu** [comment]
You're using min and max, so the value is converted to something within the interval. Using `undefined` as v-model is a user error anyway.

**Comment by alexey2baranov** [comment]
i am not shure you are right. Let suppose my client should input his appartment number. As you can imagine it is from 1 up to 9000

in clear html  we will do this as https://jsfiddle.net/ohe7m7p3/   At the initial state input is clear. But when user start typing input value never lower then min value. This is exactly what i need 
how can i get same behavior with quasar?

**Comment by rstoenescu** [comment]
Will re-analyze the component. Don't have time at the moment to take a decision but I'll take care of this in the next few days.

**Comment by alexey2baranov** [comment]
thank you again!

**Comment by rstoenescu** [comment]
Fix available in edge version and in future v0.13 (to be released really soon).
